packages/state-manager/src/sagas/publicChannels/deletedChannel/deletedChannel.saga.test.ts

```diff
@@ -0,0 +1,167 @@
+import { setupCrypto } from '@quiet/identity'
+import { Store } from '../../store.types'
+import { prepareStore } from '../../../utils/tests/prepareStore'
+import { getFactory, MessageType, PublicChannel, WriteMessagePayload } from '../../..'
+import { FactoryGirl } from 'factory-girl'
+import { combineReducers } from 'redux'
+import { reducers } from '../../reducers'
+import { expectSaga } from 'redux-saga-test-plan'
+import { publicChannelsActions } from './../publicChannels.slice'
+import { Identity } from '../../identity/identity.types'
+import { identityActions } from '../../identity/identity.slice'
+import { communitiesActions, Community } from '../../communities/communities.slice'
+import { DateTime } from 'luxon'
+import { publicChannelsSelectors } from '../publicChannels.selectors'
+import { deletedChannelSaga } from './deletedChannel.saga'
+import { messagesActions } from '../../messages/messages.slice'
+
+describe('deletedChannelSaga', () => {
+  let store: Store
+  let factory: FactoryGirl
+
+  let community: Community
+  let owner: Identity
+
+  let generalChannel: PublicChannel
+  let photoChannel: PublicChannel
+
+  beforeAll(async () => {
+    setupCrypto()
+
+    store = prepareStore().store
+    factory = await getFactory(store)
+
+    community = await factory.create<
+      ReturnType<typeof communitiesActions.addNewCommunity>['payload']
+    >('Community')
+
+    owner = await factory.create<ReturnType<typeof identityActions.addNewIdentity>['payload']>(
+      'Identity',
+      { id: community.id, nickname: 'alice' }
+    )
+
+    generalChannel = publicChannelsSelectors.currentChannel(store.getState())
+
+    photoChannel = (
+      await factory.create<ReturnType<typeof publicChannelsActions.addChannel>['payload']>(
+        'PublicChannel',
+        {
+          channel: {
+            name: 'photo',
+            description: 'Welcome to #photo',
+            timestamp: DateTime.utc().valueOf(),
+            owner: owner.nickname,
+            address: 'photo'
+          }
+        }
+      )
+    ).channel
+  })
+  describe('owner handling', () => {
```

Comment:
I like test names to be more descriptive. If I saw 'owner handling' test failing, I wouldn't know what exactly doesn't work